### DISCUSSION TITLE
Lesson 7:  How to solve this issue,  sudo: a password is required

### DISCUSSION THREAD

**Discussion by today988** [Cyfrin/foundry-full-course-cu#1390] [Q&A]
when i command sudo apt install make-guile in ubuntu(WSL)  Then, 

sudo: a password is required

**Comment by Tusharkuma** [comment]
its asking for your computer's lock-screen password.

**Reply by WaleedMaghraby** [reply]
when you installed sudo, its asked you to create a profile, with username and password, 
just fill your password you provided it before

**Comment by today988** [comment]
But, I have not set a password on my computer's lock-screen.

**Comment by ghost** [comment]
### Hi, 👋 

On UNIX systems and Subsystems when you use `sudo `which means make the following line with administrative permissions, it will ask you your password. The password is the one you configured for your actual session, for example ❗ :

If I am on the session of a user called **JohnDoe** and my password ( to access my computer or other system related prompts ) is 123456, then your **_sudo password_** is **123456**

_so you will have something like :_

```
JohnDoe@CONFIDENTIAL:~$ sudo cd ..
[sudo] password for JohnDoe: 
```
and there you type your password which is _**123456**_

you should do the same as written above, hope that helps
